src/Chat.tsx

```diff
@@ -117,6 +117,10 @@ export class Chat extends React.Component<ChatProps, {}> {
     }
 
     private saveShellRef(shellWrapper: any) {
+        if (!shellWrapper) {
+            this.shellRef = null;
+            return;
+        }
         this.shellRef = shellWrapper.getWrappedInstance();
```

Comment:
IMHO, `this.shellRef = shellWrapper && shellWrapper.getWrappedInstance();` is a better solution. But I will accept this one.